rust/connlib/tunnel/src/client.rs

```diff
@@ -1841,9 +1839,7 @@ mod proptests {
             resource_routes
                 .into_iter()
                 .chain(iter::once(IPV4_RESOURCES.into()))
-                .chain(iter::once(IPV6_RESOURCES.into()))
-                .chain(iter::once(DNS_SENTINELS_V4.into()))
-                .chain(iter::once(DNS_SENTINELS_V6.into())),
+                .chain(iter::once(IPV6_RESOURCES.into())),
```

Comment:
Quick note, this introduced a bug since our IPv6 range does _not_ include the DNS sentinels. Might as well just add the /48